river/drift/page_hinkley.py

```diff
@@ -86,12 +88,18 @@ def update(self, value) -> tuple:
 
         self.sample_count += 1
 
+        if self.sum <= self.Mint:
+            self.Mint = self.sum
+
+        pht = self.sum - self.Mint
+
         self._in_concept_change = False
+        self._in_warning_zone = False
 
         if self.sample_count < self.min_instances:
             return False, False
 
-        if self.sum > self.threshold:
```

Comment:
Also, here should be `if self.sum >= self.threshold:` according to the original paper. Am I right?

Reply:
Correct.

Reply:
I seem not able to the change the code from here. Should I close this pull request and make another one? thanks.

Reply:
That is strange. Are you getting an error? Here is some useful information: 

https://github.com/online-ml/river/blob/master/CONTRIBUTING.md

You shouldn't need to create a new PR.

Reply:
I have managed to find a way to edit it. Thanks!